content/src/service/validations/Validations.ts

```diff
@@ -119,9 +119,9 @@ export class ValidatorInstance {
     }
 
     /** Validate that the pointers are valid, and that the Ethereum address has write access to them */
-    async validateAccess(entityType: EntityType, pointers: Pointer[], ethAddress: EthAddress, validationContext: ValidationContext): Promise<void> {
+    async validateAccess(entityType: EntityType, pointers: Pointer[], timestamp: Timestamp, ethAddress: EthAddress, validationContext: ValidationContext): Promise<void> {
```

Comment:
Nit: we could pass the entire entity, instead of passing many of its properties separately